modules/phase_field/doc/content/source/postprocessors/DiscreteNucleationData.md

```diff
@@ -4,9 +4,10 @@
 
 Outputs the number of currently active nucleation sites maintained by the given
 [DiscreteNucleationInserter](/DiscreteNucleationInserter.md) (`value` option
-`COUNT`) or a boolean (0/1) value denoting if at least one new nucleus was
+`COUNT`), a boolean (0/1) value denoting if at least one new nucleus was
 inserted or removed (i.e. its hold time expired) for the current timestep
-(`value` option `UPDATE`).
+(`value` option `UPDATE`), or the total nucleation rate for the entire domain
+(`value` option `RATE`).
 
```

Comment:
The wording here is a bit confusing.  I'd suggest something like "Outputs one of three possible types of data: 1) xyz, 2) ... 3)... "